cmd/skupper/skupper.go

```diff
@@ -459,6 +466,7 @@ func NewCmdExpose(skupperCli SkupperServiceClient) *cobra.Command {
 	cmd.Flags().IntSliceVar(&(exposeOpts.Ports), "port", []int{}, "The ports to expose on")
 	cmd.Flags().StringSliceVar(&(exposeOpts.TargetPorts), "target-port", []string{}, "The ports to target on pods")
 	cmd.Flags().BoolVar(&exposeOpts.EnableTls, "enable-tls", false, "If specified, the service will be exposed over TLS (valid only for http2 and tcp protocols)")
+	cmd.Flags().StringVar(&(exposeOpts.IngressMode), "ingress-mode", "", "Controls whether a kubernetes service is created locally for the exposed service. Valid values are Always (default) or Never.")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&(exposeOpts.IngressMode), "ingress-mode", "", "Controls whether a Kubernetes service is created in the site namespace when you use Skupper to expose a service.  'Always' (default) and  'Never' are valid values.")
```
@grs I'm not sure whether my suggestion is valid? How could we describe the use case where users would use this option.
eg If you are using something like Istio, you might want to prevent Skupper from creating and clobbering services.